### PR TITLE
Update glfw package to 3.3.0-p5

### PR DESCRIPTION
**Pull request by FelikZ**
* I've followed [this guide](https://docs.hunter.sh/en/latest/creating-new/update.html)
  step by step carefully. **Yes**

* I've tested this package remotely and have excluded all broken builds.
  Here is the links to the Travis/AppVeyor with status "All passed":

  * https://ci.appveyor.com/project/FelikZ/hunter/builds/20992013
  * https://travis-ci.org/FelikZ/hunter/builds/467628640

\* linux builds are failed with the same reason as it failed in [current state](https://travis-ci.org/ingenue/hunter/jobs/467059555) ([felikz#L978](https://travis-ci.org/FelikZ/hunter/jobs/467100496#L978) vs [hunter#L988](https://travis-ci.org/ingenue/hunter/jobs/467059555#L988))
I think its being broken for a while already.

### CODE REVIEW

**Comment by FelikZ** [comment]
https://github.com/hunter-packages/glfw/pull/3

File: cmake/projects/glfw/hunter.cmake [hunk 1]
```diff
@@ -78,6 +78,17 @@ hunter_add_version(
     455c97ea01a1b78ae39a72d00d86eb1c455e0dca
 )
 
+hunter_add_version(
+    PACKAGE_NAME
+    glfw
+    VERSION
+    "3.3.0-p5"
```
Comment (ruslo) [inline cmake/projects/glfw/hunter.cmake]: Should be `3.3.0-f9923e9-p0`.

I've added section to documentation:
* https://docs.hunter.sh/en/latest/creating-new/create/cmake.html#consistency

**Comment by ruslo** [comment]
Testing:
* https://ci.appveyor.com/project/ingenue/hunter/builds/20992087
* https://travis-ci.org/ingenue/hunter/builds/467630021?utm_source=github_status&utm_medium=notification

**Comment by FelikZ** [comment]
@ruslo looks good, thanks!

**Comment by ruslo** [comment]
Uploading:
* https://ci.appveyor.com/project/ingenue/hunter/builds/20992664
* https://travis-ci.org/ingenue/hunter/builds/467640990?utm_source=github_status&utm_medium=notification

**Comment by ruslo** [comment]
Released: https://github.com/ruslo/hunter/releases/tag/v0.23.73

**Comment by FelikZ** [comment]
🥳

**Pull request merged**